public/app/plugins/datasource/graphite/specs/query_ctrl.test.ts

```diff
@@ -200,19 +202,20 @@ describe('GraphiteQueryCtrl', () => {
   });
 
   describe('when autocomplete for metric names is not available', () => {
-    silenceConsoleOutput();
+    // silenceConsoleOutput();
     beforeEach(() => {
       ctx.ctrl.state.datasource.getTagsAutoComplete = jest.fn().mockReturnValue(Promise.resolve([]));
       ctx.ctrl.state.datasource.metricFindQuery = jest.fn().mockReturnValue(
         new Promise(() => {
+          console.log('throw new Error();');
```

Comment:
is this intentional or is it a leftover?

Reply:
- [ ] revert accidental debug changes :)